unit3-server-mvc-with-node-express-and-sql/README.md

```diff
@@ -2,50 +2,58 @@
 
 ### Week 1
 
+- Serverside MVC with Express: 3 days
+  - Why MVC for server side development: Separation of concerns
+  - Routing & (simple) regular expressions
+  - Controllers
+  - Views using a templating engine and ViewModels (Mustache? Jade?)
+  - Middleware for session, error handling
+  - **Learning outcome**: Given a small existing website, add new URLs, add content to existing URLs, create a new URL with a new controller that reuses an existing view
+  - **Learning outcome**: Create a website with over 4 unique URL routes with both HTML and JSON responses, user state stored in sessions, and an interactive set of client pages using AJAX to hit at least one JSON endpoint
+- Trees & Graphs: 2 days (Carries over into the next week)
```

Comment:
Yah i think that's too complicated. I think we leave it as it is after your first suggested swap, and just tease recursive traversal in week 2 when we teach trees and graphs.